src/instruments/src/Clock/Components/Chrono.tsx

```diff
@@ -13,11 +13,14 @@ export const Chrono = () => {
     const [prevTime, setPrevTime] = useState(absTime);
 
     const [elapsedTime, setElapsedTime] = useState<null | number>(null);
+    const [elapsedSimvarTime, setElapsedSimvarTime] = useSimVar('L:A32NX_CHRONO_ELAPSED_TIME', 'number');
```

Comment:
Does not seem like we use the value itself

```suggestion
    const [, setElapsedSimvarTime] = useSimVar('L:A32NX_CHRONO_ELAPSED_TIME', 'number');
```